src/output-json-alert.c

```diff
@@ -243,6 +243,10 @@ void AlertJsonHeader(const Packet *p, const PacketAlert *pa, json_t *js)
         }
     }
 
+    if (p->flow) {
+        json_object_set_new(js, "app_proto",
```

Comment:
On doc that's what I've done in this PR.

I'm gonna try to implement the flow logging in the coming days.